src/index.html

```diff
@@ -11,12 +11,98 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
+    <header class="header-container">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
```

Comment:
The `<a>` tag for the logo currently has an `href` attribute set to `#`, which is a placeholder. Consider updating this to point to the homepage or a relevant URL to improve navigation.